docs/verify/stabilizer-circuit-simulation.ipynb

```diff
@@ -0,0 +1,199 @@
+{
```

Comment:
<blockquote>&nbsp;primitive backed the the stabilizer simulator to compute</blockquote><p>typo?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/322/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done.